airbyte-webapp/src/views/layout/SideBar/components/SidebarDropdownMenu.module.scss

```diff
@@ -0,0 +1,78 @@
+@use "../../../../scss/colors";
+@use "../../../../scss/variables";
+
+.sidebarMenu {
+  position: relative;
+}
+
+.button {
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+  justify-content: center;
+  height: 70px;
+  width: 100%;
+  background: transparent;
+  border: 0;
+  outline: none;
+  cursor: pointer;
+  border-radius: variables.$border-radius-md;
+  line-height: 15px;
+  color: colors.$grey-30;
+  text-decoration: none;
+  transition: variables.$transition-out;
+
+  .text {
+    margin-top: 7px;
+    color: colors.$grey-30;
+  }
+
+  &.open {
+    background-color: colors.$dark-blue-600;
+  }
+
+  &:hover:not(.open) {
+    background-color: colors.$dark-blue-800;
+  }
+}
+
+.items {
+  position: absolute;
+  left: 100%;
+  bottom: 0;
+  min-width: 260px;
+  margin: 0 0 14px 12px;
```

Comment:
Are these exact numbers critical? If we can, it would be better to adjust it to use one of the spacing variables.

Reply:
It's the design of the popup. In this case, we cannot use values from `variables`